specification/containerservice/resource-manager/Microsoft.ContainerService/fleet/stable/2025-03-01/fleets.json

```diff
@@ -2530,6 +2537,13 @@
           "minLength": 1,
           "maxLength": 50,
           "pattern": "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
+        },
+        "labels": {
+          "type": "object",
+          "description": "The labels for the fleet member.",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

Use of additionalProperties is not allowed for properties owned by the service. The only time its ok to use it is when the properties are pass thru (user defined) and not subject to any validations. Please make this an array or provide an explanation for why you need this.

Reply:
duplicate, see response above